### PR TITLE
Added Calender, Random Number and ASCII Programs

### PR DESCRIPTION
**Pull request by aaishikasb**
Added the following Programs

- [x]  Calculator
- [x] ASCII 
- [x] Random Number Generator

### CODE REVIEW

File: ASCIIValue.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+c = 'p'
```
Comment (Akshu-on-github) [inline ASCIIValue.py:1]: You might want to add code for user input over here

File: Calender.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import calendar
+yy = int(input("Enter year: "))
+mm = int(input("Enter month: "))
```
Comment (Akshu-on-github) [inline Calender.py:3]: You might want to mention that the user can only enter a number

**Review by Akshu-on-github — APPROVED** [review]

File: Calender.py (hunk 2 above)
Comment (aaishikasb) [reply Calender.py:3]: Will it be convenient for you to fix it since you have write access?

Comment (Akshu-on-github) [reply Calender.py:3]: Hm, sure thing

**Pull request merged**